addons/point_of_sale/static/src/js/Screens/ReceiptScreen/ReceiptScreen.js

```diff
@@ -106,6 +106,13 @@ odoo.define('point_of_sale.ReceiptScreen', function (require) {
                 const orderName = order.get_name();
                 const orderClient = { email: this.orderUiState.inputEmail, name: client ? client.name : this.orderUiState.inputEmail };
                 const order_server_id = this.env.pos.validated_orders_name_server_id_map[orderName];
+                if (!order_server_id) {
+                    this.showPopup('ErrorPopup', {
+                        title: 'Unsynced order',
+                        body: 'This order is not yet synced to server. Make sure it is synced then try again.',
```

Comment:
This should be translatable.
Make sure to also add those new term in the `.pot` file